compiler/optimizer/OMROptimizer.cpp

```diff
@@ -1566,7 +1566,6 @@ int32_t OMR::Optimizer::performOptimization(const OptimizationStrategy *optimiza
             bool blocksArePending = false;
             while (subGroup->_num != endGroup && subGroup->_num != endOpts)
                {
-               OMR::Optimizations optNum = subGroup->_num;
```

Comment:
Unless I am mistaken, this definition of optNum is "dead" in that no one seems to use this value inside the scope/loop containing the definition.